source/features/clean-repo-tabs.tsx

```diff
@@ -9,8 +9,12 @@ import getTabCount from '../github-helpers/get-tab-count';
 import looseParseInt from '../helpers/loose-parse-int';
 import {getWorkflows} from './next-scheduled-github-action';
 import abbreviateNumber from '../helpers/abbreviate-number';
-import {onlyShowInDropdown} from './more-dropdown';
 import {buildRepoURL, getRepo} from '../github-helpers';
+import {onlyShowInDropdown, unhideOverflowDropdown} from './more-dropdown';
+
+async function canUserEditOrganization(): Promise<boolean> {
+	return Boolean(await elementReady('.btn-primary[href$="repositories/new"]'));
```

Comment:
This only works on the Repositories tab. Refer to *both* the screenshots I posted or visit https://github.com/rollup and look for items in each tab. There probably isn't just one for everyone and maybe on some tabs there's nothing at all, so let's just try to match as many as possible.

I'm looking for something like:
```suggestion
	return Boolean(await elementReady([
		'.btn-primary[href$="repositories/new"]', // repo tab, or isOrgRepos if it exists
		'other button', // other tab,
		'etc
```

Reply:
I think you missed my post in the other PR.

There are no counts on any other pages. So we cannot hide anything.

Reply:
Which makes me wonder, why are we naming this `clean-REPO-tabs` when it also covers organizations? And why doesn't it cover regular users?

Reply:
Anyway yes I missed that message. And yuck every tab is different on there 😂 

![gif](https://user-images.githubusercontent.com/1402241/126010614-c1b0bcc2-63d7-48e3-ade9-be6611ba1394.gif)

Reply:
Its only one tab in organizations. We could drop it altogether.



> And why doesn't it cover regular users?

Not sure what you mean here?